src/Test/Utilities/Portable/MarkedSource/MarkupTestFile.cs

```diff
@@ -58,11 +58,12 @@ public static class MarkupTestFile
 
             // A stack of span starts along with their associated annotation name.  [||] spans simply
             // have empty string for their annotation name.
-            var spanStartStack = new Stack<Tuple<int, string>>();
+            var spanStartStack = new Stack<(int, string)>();
+            var namedSpanStartStack = new Stack<(int, string)>();
```

Comment:
Why this change?

Reply:
Less allocation, old class-based tuples are slowly being replaced where possible.

Reply:
Ability to use names instead of Element1 and Element2 (see new commit).

Reply:
Sorry, i meant specifically: why the switch to two stacks?

Reply:
So that the named & not named can interleave each other. E.g.:
```
        //[|>
        var str = {|result:"" <|] aaa""|};
```

Previously only nesting was allowed

Reply:
terrifying!  but ok :D